### PR TITLE
Miscellaneous clean-ups

### PR DESCRIPTION
**Pull request by owen-mc**

### CODE REVIEW

File: ql/src/semmle/go/frameworks/stdlib/EncodingJson.qll [hunk 1]
```diff
@@ -104,14 +101,6 @@ module EncodingJson {
       hasQualifiedName("encoding/json", "Encoder", "SetIndent") and
       (inp.isParameter(_) and outp.isReceiver())
       or
-      // signature: func (RawMessage).MarshalJSON() ([]byte, error)
-      hasQualifiedName("encoding/json", "RawMessage", "MarshalJSON") and
```
Comment (smowton) [inline ql/src/semmle/go/frameworks/stdlib/EncodingJson.qll:108]: Why were these removed?

Comment (owen-mc) [reply ql/src/semmle/go/frameworks/stdlib/EncodingJson.qll:108]: It is explained in the commit message:
` Remove redundant code

RawMessage implements Marshaler and Unmarshaler, so these methods are
covered by the two sections below`

**Review by smowton — APPROVED** [review]

File: ql/src/semmle/go/frameworks/stdlib/EncodingJson.qll (hunk 1 above)
Comment (smowton) [reply ql/src/semmle/go/frameworks/stdlib/EncodingJson.qll:108]: Assuming you've verified that works as intended then, lgtm
